support/tarball/build.sh

```diff
@@ -12,5 +12,25 @@ export NUGET_PACKAGES="$SCRIPT_ROOT/packages/"
 
 MSBUILD_ARGUMENTS=("/p:OfflineBuild=true" "/flp:v=detailed" "/clp:v=detailed")
 
+echo "Expanding BuildTools dependencies into packages directory..."
+# init-tools tries to copy from its script directory to Tools, which in this case is a copy to
+# itself. This is an error. To avoid the error, use a temp dir that we immediately delete.
+TEMP_TOOLS_DIR="$SCRIPT_ROOT/ToolsTemp"
+PREBUILT_PACKAGE_SOURCE="$SCRIPT_ROOT/prebuilt/nuget-packages"
+(
+    set -x
```

Comment:
Did you mean to leave this here?

Reply:
Yeah, this makes the commands show up in the logs without the maintenance cost and risk of duplicating the commands in `echo ...` statements.